src/Compilers/CSharp/Portable/Symbols/Compilation_WellKnownMembers.cs

```diff
@@ -300,7 +300,9 @@ internal static Symbol GetRuntimeMember(NamedTypeSymbol declaringType, ref Membe
             ImmutableArray<TypedConstant> arguments = default(ImmutableArray<TypedConstant>),
             ImmutableArray<KeyValuePair<WellKnownMember, TypedConstant>> namedArguments = default(ImmutableArray<KeyValuePair<WellKnownMember, TypedConstant>>))
         {
-            var ctorSymbol = (MethodSymbol)GetWellKnownTypeMember(constructor);
+            DiagnosticInfo useSiteDiagnostics = null;
```

Comment:
Please rename to `diagnosticInfo` (singular). And the initializer is unnecessary.